src/presenters/gouvernancePresenter.ts

```diff
@@ -0,0 +1,218 @@
+import { formaterEnDateFrancaise } from './shared/date'
+import { formaterEnNombreFrancais } from './shared/number'
+import { ComiteReadModel, FeuilleDeRouteReadModel, MembreReadModel, UneGouvernanceReadModel } from '@/use-cases/queries/RecupererUneGouvernance'
+
+export function gouvernancePresenter(
+  gouvernanceReadModel: UneGouvernanceReadModel
+): GouvernanceViewModel {
+  return {
+    ...{ comites: gouvernanceReadModel.comites?.map(toComitesViewModel) },
+    departement: gouvernanceReadModel.departement,
+    isVide: isGouvernanceVide(gouvernanceReadModel),
+    sectionFeuillesDeRoute: {
+      ...{ feuillesDeRoute: gouvernanceReadModel.feuillesDeRoute?.map(toFeuillesDeRouteViewModel) },
+      ...buildTitresFeuillesDeRoute(gouvernanceReadModel.feuillesDeRoute),
+    },
+    sectionMembres: {
+      ...{ membres: gouvernanceReadModel.membres?.map(toMembresViewModel) },
+      ...buildSousTitreMembres(gouvernanceReadModel.membres),
+    },
+    sectionNoteDeContexte: {
+      ...{ noteDeContexte: toNoteDeContexteViewModel(gouvernanceReadModel.noteDeContexte) },
+      sousTitre: buildSousTitreNoteDeContexte(gouvernanceReadModel.noteDeContexte),
+    },
+  }
+}
+
+export type GouvernanceViewModel = Readonly<{
+  comites?: ReadonlyArray<ComiteViewModel>
+  departement: string
+  isVide: boolean
+  sectionFeuillesDeRoute: Readonly<{
+    budgetTotalCumule: string
+    feuillesDeRoute?: ReadonlyArray<FeuilleDeRouteViewModel>
+    total: string
+    wording: string
+    lien: Readonly<{
+      label: string
+      url: URL
+    }>
+  }>
+  sectionMembres: Readonly<{
+    detailDuNombreDeChaqueMembre: string
+    membres?: ReadonlyArray<MembreViewModel>
+    total: string
+    wording: string
+  }>
+  sectionNoteDeContexte: Readonly<{
+    noteDeContexte?: NoteDeContexteViewModel
+    sousTitre: string
+  }>
+}>
+
+function isGouvernanceVide(gouvernanceReadModel: UneGouvernanceReadModel): boolean {
+  return !gouvernanceReadModel.comites &&
+    !gouvernanceReadModel.membres &&
+    !gouvernanceReadModel.feuillesDeRoute &&
+    !gouvernanceReadModel.noteDeContexte
```

Comment:
```suggestion
  return [
    gouvernanceReadModel.comites,
    gouvernanceReadModel.membres,
    gouvernanceReadModel.feuillesDeRoute,
    gouvernanceReadModel.noteDeContexte
  ].every(isNullish)
```

Ça tue les mutants , qui n'ont plus lieu d'être sur de la logique booléenne, et surtout, c'est beaucoup déclaratif, et ça se lit comme "si tous ces éléments sont absents".